sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -63,4 +68,92 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
       count: applications.count,
     };
   }
+
+  /**
+   * Gets all in progress application offering request changes.
```

Comment:
Can we please change `Gets` to `Get` to keep it in sync with other file definition comments.

Reply:
I can change it, but when u say to make it in sync, we do have these many files using the same, 
![image](https://github.com/bcgov/SIMS/assets/77353155/2ee49ae2-1269-4d58-aeaf-746113ff4ae6)

Reply:
I personally prefer "gets" because when you are reading the comment for the method you usually do something: "Method x does y" or "Method A gets 'xyz'".

Reply:
It is a nice split. Many methods have this type of comment pattern and all others have the other type. Resolving this since it is very minor.